packages/canary-features/addon/default-features.ts

```diff
@@ -173,10 +173,8 @@ export default {
   SAMPLE_FEATURE_FLAG: null,
   RECORD_DATA_ERRORS: true,
   RECORD_DATA_STATE: true,
-  IDENTIFIERS: true,
```

Comment:
I think we might want to keep these, in case someone was reading them and branching code off of it, removing them would break. Do you think there is a downside in not GCing them?

Reply:
I only thought to commit these for clarity of the codebase.  Certainly if someone was relying on a forked version of e-d, but is it something we expect users to read?  And if they do, then it is expected they could be removed at any point?  As long as we get the tests passing, I'm ok to do whatever you think is best 👍